content/guides/core-concepts/writing-and-organizing-tests.md

```diff
@@ -259,36 +315,42 @@ beforeEach(() => {
 #### Execution
 
 Cypress executes the support file before the spec file. For example when you
-click on a test file named `spec-a.js` via
+click on a test file named `spec-a.cy.js` via
 [cypress open](/guides/guides/command-line#cypress-open), then the Cypress App
 executes the files in the following order:
 
-```html
-<!-- bundled support file -->
-<script src="support/index.js"></script>
-<!-- bundled spec file -->
-<script src="integration/spec-a.js"></script>
-```
+**e2e example:**
+
+1. `support/e2e.js // your support file`
+2. `e2e/spec-a.cy.js // your spec file`
```

Comment:
How about formatting more like this?

```suggestion
1. `support/e2e.js` (your support file)
2. `e2e/spec-a.cy.js` (a spec file)
```

Reply:
This commit changes the formatting via your feedback https://github.com/cypress-io/cypress-documentation/pull/4328/commits/ae9abc26bccc4b33ed054dd3806b733aafc2400b